### PR TITLE
chore(documentation-v7): merge storybook v7 branches

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

File: packages/documentation-v7/.storybook/main.ts [hunk 1]
```diff
@@ -9,17 +9,22 @@ const config: StorybookConfig = {
...
   },
-  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(ts|tsx)'],
+  stories: ['../src/stories/**/*.mdx', '../src/stories/**/*.stories.@(ts|tsx)'],
   addons: [
-    // essential addons
-    '@storybook/addon-essentials',
-
-    // community addons
-    'storybook-dark-mode',
-    // '@pxtrn/storybook-addon-docs-stencil',
```
Comment (imagoiq) [inline packages/documentation-v7/.storybook/main.ts:19]: Maybe we should keep this one commented to not forget? Or is it intentional?

File: packages/documentation-v7/tsconfig.json [hunk 2]
```diff
@@ -15,6 +15,11 @@
...
   },
-  "include": ["src"],
-  "exclude": ["node_modules"]
+  "include": [
+    ".storybook/**/*",
+    "src/**/*"
+  ],
+  "exclude": [
+    "node_modules/**/*"
+  ]
```
Comment (imagoiq) [inline packages/documentation-v7/tsconfig.json:24]: Good, I think it resolves an issue that I saw today with the current doc.

**Review by imagoiq — APPROVED** [review]

File: packages/documentation-v7/.storybook/blocks/docs/layout.tsx [hunk 3]
```diff
@@ -1,24 +1,30 @@
-import React from 'react';
-
-import { DocsContainer as Layout, Unstyled } from '@storybook/blocks';
-import Header from './header';
+import { DocsContainer, DocsContainerProps, Unstyled } from '@storybook/blocks';
+import React, { PropsWithChildren } from 'react';
+import { useDarkMode } from 'storybook-dark-mode';
```
Comment (oliverschuerch) [inline packages/documentation-v7/.storybook/blocks/docs/layout.tsx]:
```suggestion
```

File: packages/documentation-v7/.storybook/blocks/docs/layout.tsx [hunk 4]
```diff
@@ -1,24 +1,30 @@
...
-      {showFooter() ? <Footer /> : null}
-    </Unstyled>
-  </Layout>
-);
+export default ({ children, context }: PropsWithChildren<DocsContainerProps>) => {
+  return (
+    <DocsContainer
+      context={context}
+      theme={useDarkMode() ? themes.dark : themes.light}
+    >
```
Comment (oliverschuerch) [inline packages/documentation-v7/.storybook/blocks/docs/layout.tsx]:
```suggestion
    <DocsContainer context={context}>
```

Togge the darkmode is working properly without this.
https://storybook.js.org/addons/storybook-dark-mode/

File: packages/documentation-v7/.storybook/main.ts [hunk 5]
```diff
@@ -9,17 +9,22 @@ const config: StorybookConfig = {
...
     '@storybook/addon-links',
+    '@storybook/addon-a11y',
```
Comment (oliverschuerch) [inline packages/documentation-v7/.storybook/main.ts:15]: We should avoid installing unnecessary modules.
We can add this module, when we really use it.
For now, I would only leave it as a comment (for example).

File: packages/documentation-v7/package.json [hunk 6]
```diff
@@ -31,10 +31,12 @@
   },
   "devDependencies": {
     "@geometricpanda/storybook-addon-badges": "1.1.1",
+    "@storybook/addon-a11y": "6.5.16",
```
Comment (oliverschuerch) [inline packages/documentation-v7/package.json:34]:
```suggestion
```

File: packages/documentation-v7/package.json [hunk 7]
```diff
@@ -31,10 +31,12 @@
...
     "@storybook/addon-essentials": "7.0.26",
     "@storybook/addon-links": "7.0.26",
     "@storybook/blocks": "7.0.26",
     "@storybook/manager-api": "^7.0.26",
+    "@storybook/testing-library": "0.0.14-next.2",
```
Comment (oliverschuerch) [inline packages/documentation-v7/package.json:39]:
```suggestion
```
It does not make any sense to install it, but not to use it in the main.ts file.
And this addon only works together with some other modules, which are not installed yet:
https://storybook.js.org/docs/web-components/writing-tests/interaction-testing

File: packages/documentation-v7/src/stories/foundation/elevation/elevation.docs.mdx [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+/>
+
+# Elevation
+
+<p className="lead">Elevation uses box shadows to give the impression of a distance between two elements along the z axis.</p>
+
+The higher the elevation, the larger and softer the shadow, expressing a greater distance between an element and the one behind it.
+
+<div className="elevation-container">
+  {[1, 2, 3, 4, 5].map((elevationLevel, i) => (
```
Comment (oliverschuerch) [inline packages/documentation-v7/src/stories/foundation/elevation/elevation.docs.mdx:18]: We could get the amount of elevation levels from a elevation.module.scss.
What do you think?

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/documentation-v7/.storybook/main.ts (hunk 5 above)
Comment (alizedebray) [reply packages/documentation-v7/.storybook/main.ts:15]: This addon was explicitly added with PR #979 so I think we will need to have it anyway.
I've commented it out for now so we don't forget to add it later.

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**